utils/progressbar/transferfileprogressbar.go

```diff
@@ -134,6 +135,10 @@ func (t *TransferProgressMng) RemoveRepository() {
 	time.Sleep(ProgressRefreshRate)
 }
 
+func (t *TransferProgressMng) StopGracefully() {
```

Comment:
Might need to hold the bar returned from this function, and abort it with drop=true in `Quit()`. See how we handle other headlines and check if it is necessary.